MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+# Migration from Splunk Connect for Kubernetes
+
+## Assumptions
+
+You are running Splunk Connect for Kubernetes (SCK) 1.4.9 and want to migrate to Splunk OpenTelemetry Collector for Kubernetes. Previous versions of the SCK are out of scope, but this guide still applies. You might encounter some version specific issues (such as missing/changed config options) but that shouldn't prevent you from proceeding with migration.
+
+## Changes in Components
+
+Splunk Connect for Kubernetes(SCK) has 3 components/applications:
+1.  Logs, metrics and traces from a kubernetes cluster (deployed as a DaemonSet)
+2.  Application to fetch cluster metrics from a kubernetes cluster (deployed as a deployment)
+3.  Application to fetch kubernetes objects metadata from a kubernetes cluster (deployed as a deployment)
+
+All the applications in SCK use Fluentd to work with logs, metrics and objects. Fluentd has significant performance issues when used to fetch logs in a Kubernetes cluster with pods that have very high throughput.  
+
+Splunk OpenTelemetry Collector for Kubernetes provides significant performance improvements over the SCK through use of the OpenTelemetry Collector agent and native OpenTelemetry functionality for logs collection rather than Fluentd. You can learn more about the performance characteristics of this new application [here](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md#performance-of-native-opentelemetry-logs-collection).
+
+Splunk OpenTelemetry Collector for Kubernetes has 2 components/applications:
```

Comment:
"Splunk OpenTelemetry Collector for Kubernetes has the following components and applications:"